docs/_docs/installation/ubuntu.md

```diff
@@ -6,7 +6,7 @@ Before we install Jekyll, we need to make sure we have all the required
 dependencies.
 
 ```sh
-sudo apt-get install ruby ruby-dev build-essential
+sudo apt-get install ruby-full ruby-dev build-essential
```

Comment:
ruby-dev is included in [ruby-full](https://packages.ubuntu.com/cosmic/ruby-full), so we can remove it here